Kickstarter.xcodeproj/xcshareddata/xcschemes/Kickstarter-Framework-iOS.xcscheme

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <Scheme
-   LastUpgradeVersion = "1200"
+   LastUpgradeVersion = "1220"
```

Comment:
What changed this?

Reply:
Ah yeah just the Xcode version I was using I guess, I can revert but this is pretty innocuous.